.github/workflows/pull-request.yaml

```diff
@@ -101,5 +101,6 @@ jobs:
         with:
           juju-channel: latest/stable
           provider: microk8s
+          channel: 1.23/stable
```

Comment:
This [might be fixed](https://github.com/charmed-kubernetes/actions-operator/pull/38) now. Wanna give 1.24 another chance?